packages/components/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## Unreleased
 
 ### Bug Fix
-
+-   `InputControl`: Fix sample code on InputControl docs [#2610](https://github.com/WordPress/gutenberg/pull/59517)
```

Comment:
```suggestion
-   `InputControl`: Fix sample code on InputControl docs [#59517](https://github.com/WordPress/gutenberg/pull/59517)
```

Reply:
CHANGELOG has been updated to reflect link to PR, not the issue